doc/jetty-config.md

```diff
@@ -129,7 +129,8 @@ virtual cores on the host divided by 8, with a minimum of 1 and maximum of 4.
 This sets the number of threads that the webserver will dedicate to processing
 events on connected sockets for _encrypted_ HTTPS traffic.  Defaults to the
 number of virtual cores on the host divided by 2, with a minimum of 1 and
-maximum of 4.
+maximum of 4. The number of selectors used by Jetty is twice the number of ssl
```

Comment:
Looking at [this code segment](https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java#L403-L408) that you linked, I'm interpreting this to mean that Jetty will allocate 2 threads for each `selector` we request it to use.  We are passing in the value in the config for `selector-threads` and `ssl-selector-threads` as the `selector` argument.  In retrospect, then, I wish we'd just named these config arguments `selectors` and `ssl-selectors`.  I guess that rather than changing the config option name, though, we should say something like this:

 > This sets the number of selectors that the webserver will dedicate to processing events on connected sockets for _encrypted_ HTTPS traffic.  Defaults to the number of virtual cores on the host divided by 2, with a minimum of 1 and maximum of 4.  The number of selector threads actually used by Jetty is twice the number of selectors requested.  For example, if a value of 3 is specified for the `ssl-selector-threads` setting, Jetty will actually use 6 selector threads.